components/bot/index.js

```diff
@@ -1,59 +1,75 @@
-import Image from 'next/image'
-import styles from "./index.module.scss"
-import {default as cards} from './bot-cards-info.json'
-import Layout from 'components/layout'
+import Image from "next/image";
+import styles from "./index.module.scss";
+import { default as cards } from "./bot-cards-info.json";
+import Layout from "components/layout";
```

Comment:
Check if is possible to fix the `.editorconfig`, or your personal code style for this project. 
(will be easy for us without conflicts in code 😄)

Reply:
Yeap! I've added it, should be all good now.